src/journal_manager/log_write/log_write_handler.cpp

```diff
@@ -117,11 +117,6 @@ LogWriteHandler::AddLog(LogWriteContext* context)
         {
             metaUpdateCb->SetLogGroupId(groupId);
         }
-        else
-        {
-            POS_TRACE_ERROR(POS_EVENT_ID::JOURNAL_CALLBACK_FAILED,
-                "meta update callback was not updated correctly");
```

Comment:
metaUpdateCb 가 nullptr 가 되는 경우도 정상인 상황에서 발생할 수 있다고 하면, error log는 삭제하는 대신 comment 로 간단히 맥락을 적어주는 것도 좋을 것 같습니다. 아래 정도는 어떨까요??
```c++
else
{
  // metaUpdateCb is nullable by design since certain callback (e.g., VolumeDeletedLogWriteCallback) 
  // may not want to use VersionedSegmentContext feature (hence, not inheriting MetaUpdateCallback)
}
```
혹시 제가 의도를 잘못 적었을 수 있으니 참고만 하시길 ^^;

Reply:
엇! 추가해주신 코멘트도 넣어서 다시 올리겠습니다. ㅋ